### PR TITLE
`Development`: Remove redundant database columns from online course configuration

### PR DESCRIPTION
**Pull request by basak-akan**
> [!WARNING]  
> **DO NOT DEPLOY CONTAINS DATABASE CHANGES!!!** 

#### General

- [x] This is a small issue that I tested locally and was confirmed by another developer on a test server.
- [x] Language: I followed the [guidelines for inclusive, diversity-sensitive, and appreciative language](https://docs.artemis.cit.tum.de/dev/guidelines/language-guidelines/).
- [x] I chose a title conforming to the [naming conventions for pull requests](https://docs.artemis.cit.tum.de/dev/development-process/#naming-conventions-for-github-pull-requests).

### Motivation and Context

With  #7620, registration_id, client_id, authorization_uri, jwk_set_uri, token_uri columns are moved to new table and not need on online course management.

### Description

This PR removes redundant columns (registration_id, client_id, authorization_uri, jwk_set_uri, token_uri) from online course management table.

### Steps for Testing

Prerequisites:
> [!WARNING]  
> **DO NOT DEPLOY CONTAINS DATABASE CHANGES!!!** 

1. Run Artemis locally 
2. Check if server starts
3. Check if the columns are removed from online_course_configuration table

#### Code Review
- [x] Code Review 1
- [x] Code Review 2
#### Manual Tests
- [x] Test 1
- [x] Test 2

### CODE REVIEW

**Review by muradium — DISMISSED** [review]
Tested in testing session, changelog makes expected changes

**Review by egekurt123 — DISMISSED** [review]
Tested locally, works as expected

**Review by beyzaaltuntas — DISMISSED** [review]
Tested locally, works as expected

File: src/main/resources/config/liquibase/master.xml [hunk 1]
```diff
@@ -88,6 +88,7 @@
     <include file="classpath:config/liquibase/changelog/20240115231508_changelog.xml" relativeToChangelogFile="false"/>
     <include file="classpath:config/liquibase/changelog/20240118175408_changelog.xml" relativeToChangelogFile="false"/>
     <include file="classpath:config/liquibase/changelog/20240307174511_changelog.xml" relativeToChangelogFile="false"/>
+    <include file="classpath:config/liquibase/changelog/20240304125234_changelog.xml" relativeToChangelogFile="false"/>
```
Comment (b-fein) [inline src/main/resources/config/liquibase/master.xml:91]:
```suggestion
    <include file="classpath:config/liquibase/changelog/20240304125234_changelog.xml" relativeToChangelogFile="false"/>
    <include file="classpath:config/liquibase/changelog/20240307174511_changelog.xml" relativeToChangelogFile="false"/>
```
swap for chronological order, changelogs should change separate tables and not affect each other

**Review by b-fein — DISMISSED** [review]

**Review by egekurt123 — APPROVED** [review]
Reapprove

**Review by b-fein — APPROVED** [review]

**Review by beyzaaltuntas — APPROVED** [review]

**Review by muradium — APPROVED** [review]

**Pull request merged**